packages/gestalt/src/ModuleExpandable.js

```diff
@@ -28,43 +22,50 @@ export default function ModuleExpandable({
   id,
   items,
   onExpandedChange,
-}: Props): Node {
+}: PublicModuleExpandableProps): Node {
   const [expandedId, setExpandedId] = useState<?number>(getExpandedId(expandedIndex));
 
   useEffect(() => {
     setExpandedId(getExpandedId(expandedIndex));
   }, [expandedIndex, setExpandedId]);
 
+  const onModuleClickedHandler = useCallback(
```

Comment:
nit: this is a function to build the handler rather than the handler itself, so something like `buildOnModuleClickHandler` would be more accurate